kani-driver/src/call_goto_instrument.rs

```diff
@@ -37,6 +37,10 @@ impl KaniSession {
             self.goto_sanity_check(output)?;
         }
 
+        for function in &harness.contracts {
```

Comment:
Please add a test (or create an issue) where multiple contracts are reachable from the same harness

Reply:
I've removed this capability, because CBMC can't deal with this anyway